src/main/nsis/samples/conf/process-conf.xml

```diff
@@ -34,12 +34,12 @@
         <property name="name" value="opportunityUpsertProcess"/>
         <property name="configOverrideMap">
             <map>
-                <entry key="sfdc.endpoint" value="https://login.salesforce.com"/> 
+                <entry key="sfdc.endpoint" value="https://login.salesforce.com"/>
```

Comment:
Is it actually necessary to let users set this value? It's a risk; I was able to steal credentials (when the real file is world-writable) but pointing it at a server I control.

Reply:
Yes. We need that.  That is also for custom domain.